app/test_split_integer.py

```diff
@@ -2,20 +2,37 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    parts = 5
+    result = split_integer(value, parts)
+    assert sum(result) == value
```

Comment:
```suggestion
    assert 10 ==  sum(split_integer(10, 5))
```

no need create variable in this case